Translation corrections (#325)
07th-mod/higurashi-console-arcs: Update/kake_086.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>レポーター</color>", NULL, "<color=#f5e6d3>Reporter</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s20/00/vtk2_repo2011", 256, TRUE);
	OutputLine(NULL, "「なにしろ今回、プラシルの副作用で病状が悪化した患者の数は、現状確認できるだけでも５００人以上…」",

> 			NULL, "\"After ll, at present, it has been confirmed that at least over 500 patients' conditions worsened due to side effects of Placil, a number that is still rising...\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"After all, at present, it has been confirmed that over 500 patients' conditions have worsened due to side effects of Placil, a number that is still rising...\"", Line_Normal);
```
